aztk/models.py

```diff
@@ -23,6 +23,13 @@ def __init__(self, name: str = None, script: str = None, run_on=None):
         self.run_on = run_on
 
 
+class UserConfiguration:
+    def __init__(self, username: str, ssh_key: str = None, password: str = None):
```

Comment:
Will this get auto-populated from the users' secrets.yaml?

Reply:
On the CLI, yeah -- that happens here: 
https://github.com/jafreck/aztk/blob/06f85c2538db98e168fed8c38fe9b88524f95e5f/cli/spark/endpoints/cluster/cluster_create.py#L87

For the SDK, no the secrets.yaml file is not used at all.